components/AddFormDialog/AddFormDialog.spec.tsx

```diff
@@ -24,11 +25,13 @@ describe('AddFormDialog', () => {
   it('shows forms from two sources', () => {
     render(
       <AuthProvider user={mockedUser}>
-        <AddFormDialog
-          isOpen={true}
-          onDismiss={jest.fn()}
-          person={mockedResident}
-        />
+        <FeatureFlagProvider features={{}}>
```

Comment:
Add additional tests around the feature flag and eventually the Google group.

Reply:
Added tests and will only display if feature flag is on and user is in workflow pilot.